config/initializers/figaro.rb

```diff
@@ -0,0 +1,2 @@
+Figaro.require_keys("GITHUB_KEY", "GITHUB_SECRET",
+                    "glassdoor_partner_id", "glassdoor_key")
```

Comment:
What's this for? I don't think I've ever had to use this before for Figaro, what is this doing?